app/views/install_stable.scala.txt

```diff
@@ -237,7 +237,8 @@ echo "sdkman_curl_max_time=10" >> "$sdkman_config_file"
 echo "sdkman_debug_mode=false" >> "$sdkman_config_file"
 echo "sdkman_insecure_ssl=false" >> "$sdkman_config_file"
 echo "sdkman_rosetta2_compatible=false" >> "$sdkman_config_file"
-echo "sdkman_selfupdate_enable=true" >> "$sdkman_config_file"
+echo "sdkman_auto_update=true" >> "$sdkman_config_file"
```

Comment:
It would be great if we could migrate the `sdkman_selfupdate_enable` config to `sdkman_auto_update`. We wouldn't want people to lose their config because we changed the name.